thunder/__init__.py

```diff
@@ -621,8 +621,6 @@ def get_computation_and_inputs(*args, **kwargs):
             # See https://github.com/Lightning-AI/lightning-thunder/issues/433
             if cd.use_cudagraphs:
                 comp = CUDAGraphExecutor(comp)
-                if backward_fn is not None:
-                    backward_fn = CUDAGraphExecutor(backward_fn, num_constant_args=len(backward_trc.args[0][0]))
```

Comment:
Will you restore this code?

Reply:
Yes, in a follow-up stack PRs.

Reply:
Is there a way to not disable functionality in one piece of the stack? What tests fail?

Reply:
It is not meant to touch this part just now, doing so for testing purposes. The correct answer to your top question is "it needs modifications". This PR, however, will not touch it just yet.